src/style.css

```diff
@@ -0,0 +1,104 @@
+* {
+  outline: 0 !important;
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  border: 0;
+  width: 100%;
+  font-size: 16px;
+  line-height: 22px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  color: #3d4e61;
+}
+
+.input:hover {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 400;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.icon {
+  position: absolute;
+  margin-top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--small {
+  border: 0;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 21px;
+  padding: 12px 20px 11px 33px;
+  color: #3d4e61;
+  width: 100%;
+}
+
+.input--small:hover {
+  outline: 0;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 400;
+}
+
+.input--small:focus {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff, #f6f6f7 100%);
+  border-radius: 3px;
+}
```

Comment:
you repeat alot of styles with your small class look through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and pay attention to 2nd item, use one class for shared styles

Reply:
done